ui/components/app/confirm-page-container/confirm-page-container.component.js

```diff
@@ -203,3 +229,42 @@ export default class ConfirmPageContainer extends Component {
     );
   }
 }
+
+function mapStateToProps(state, ownProps) {
```

Comment:
As far as I know we tend to put the "Container" in a separate file and connect/wrap the component there. You can see this pattern throughout the code base, but for just one example look at `add-token.component.js` and `add-token.container.js`. Alternatively I think we generally want, when we have the bandwidth, to migrate the components to all functional, and use the useDispatch and useSelector hooks for interacting with redux.

Reply:
This component in particular would be an easy win to convert to a functional component. Right now the only place this component is called is confirm-transaction-base.component which is passing in all of its many props.

Reply:
i'm fine with the separation into .container as well :D

Reply:
@adonesky1 I've separated the container and the component in a latest commit. Thanks.